hardware/AMEL/samd/variants/AMEL_SmartEverything/variant.cpp

```diff
@@ -355,3 +357,43 @@ void setStepUp(uint32_t on) {
         digitalWrite(PIN_REG_ON, LOW);
     }
 }
+
+
+void gpsForceOn(void){
+    bool ret = false;
+    uint8_t delay=0;
+    uint8_t data = 0;
+    
+	// Activate force on moving low
+    data = readRegister(TCA6416A_ADDRESS, OUTPUT_PORT_1);
+    data &= ~GPS_FORCE_ON_PIN;
+    
+	writeRegister(TCA6416A_ADDRESS, OUTPUT_PORT_1, data);
+
+}
+void sfxSleep(void){
+    bool ret = false;
+    uint8_t delay=0;
+    uint8_t data = 0;
+    
+	// read the IoExtender data actual situation
+    data = readRegister(TCA6416A_ADDRESS, OUTPUT_PORT_1);
+    data |= SFX_WAKEUP_PIN; // Put SFX in Sleep
+    
+	// send to IoExtender the new data
+	writeRegister(TCA6416A_ADDRESS, OUTPUT_PORT_1, data);
+
+}
+
+void sfxWakeup(void){
```

Comment:
The idea is to have all the libraries independent from the ASME,
Since are UART/I2C based, they could be reused from any other Arduino product that want to use the same chip.
But about the SFX powerSafe feature, the pin is directly connected to an internal IoExtender, so in this case if pure ASME and not reusable, the library will call this function that the other unit with Telit chip has to redefine.